core/server/services/rss/generate-feed.js

```diff
@@ -71,32 +72,33 @@ generateItem = function generateItem(post, siteUrl, secure) {
  * @param {{title, description, safeVersion, secure, posts}} data
  */
 generateFeed = function generateFeed(baseUrl, data) {
-    var siteUrl = urlService.utils.urlFor('home', {secure: data.secure}, true),
-        feedUrl = urlService.utils.urlFor({relativeUrl: baseUrl, secure: data.secure}, true),
-        feed = new RSS({
-            title: data.title,
-            description: data.description,
-            generator: 'Ghost ' + data.safeVersion,
-            feed_url: feedUrl,
-            site_url: siteUrl,
-            image_url: urlService.utils.urlFor({relativeUrl: 'favicon.png'}, true),
-            ttl: '60',
-            custom_namespaces: {
-                content: 'http://purl.org/rss/1.0/modules/content/',
-                media: 'http://search.yahoo.com/mrss/'
-            }
-        });
-
-    data.posts.forEach(function forEach(post) {
-        var item = generateItem(post, siteUrl, data.secure);
-
-        filters.doFilter('rss.item', item, post).then(function then(item) {
-            feed.item(item);
-        });
+    const {urlFor} = urlService.utils;
```

Comment:
Function `urlService.utils.urlFor` doesn't use `this`, so it can be destructured safely to optimize repeating property lookups. Anyway I've reverted to the dot notation.